src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,117 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXMotor;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXSignal;
+import frc.trigon.robot.hardware.simulation.FlywheelSimulation;
+import frc.trigon.robot.utilities.mechanisms.SpeedMechanism2d;
+
+public class ShooterConstants {
+    private static final int
+            RIGHT_MOTOR_ID = 0,
+            LEFT_MOTOR_ID = 1;
+    private static final String
+            RIGHT_MOTOR_NAME = "rightShootingMotor",
+            LEFT_MOTOR_NAME = "leftShootingMotor";
+    static final TalonFXMotor
+            RIGHT_MOTOR = new TalonFXMotor(RIGHT_MOTOR_ID, RIGHT_MOTOR_NAME, RobotConstants.CANIVORE_NAME),
+            LEFT_MOTOR = new TalonFXMotor(LEFT_MOTOR_ID, LEFT_MOTOR_NAME, RobotConstants.CANIVORE_NAME);
+    private static final InvertedValue
```

Comment:
I though we said we were having a blank line between Motor Declaration and Motor settings. Though we don't have to do this